src/web3/networkConfig.ts

```diff
@@ -167,6 +173,12 @@ const CAB_ISSUE_VALORA_KEYSHARE_MAINNET = `${CLOUD_FUNCTIONS_MAINNET}/issueValor
 const networkConfigs: { [testnet: string]: NetworkConfig } = {
   [Testnets.alfajores]: {
     networkId: '44787',
+    networkToNetworkId: {
+      [Network.Celo]: NetworkId['celo-alfajores'],
+      [Network.Ethereum]: NetworkId['ethereum-sepolia'],
+    },
+    supportedNetworkIds: [NetworkId['celo-alfajores'], NetworkId['ethereum-sepolia']],
```

Comment:
These two fields - `networkToNetworkId` and `supportedNetworkIds` - may seem a bit redundant. If we only ever "support" the networks that the wallet knows about (via the `Network` enum), then we can just use `Object.values(networkToNetworkId)` in place of `supportedNetworkIds`. `networkToNetworkId` however provides type safety (forcing us to update it when a new `Network` value is added, which is good), so this would mean that without `supportNetworkIds`, we couldn't update the `Network` enum without immediately supporting that network (or remove support for a network without modifying the `Network` enum).

Reply:
don't we filter supportedNetworkIds with the remote config before using them anyway?

Reply:
I'm not sure I see the purpose of supportedNetworkIds still

Reply:
the naming is wonky. `supportedNetworkIds` is supposed to encompass "all" the networks we support for multi-chain, assuming multi-chain features are enabled. it's not really important though; i've updated the code to just use `Object.values(networkToNetworkId)` instead.